extra/utils/test.js

```diff
@@ -0,0 +1,43 @@
+let d;
+let permissions =
+{
+    'getUsers': {
+    all: ['head-trainer'],
+    read : ['trainee', 'trainer'],
+    write : ['trainer'],
+    Delete: [],
+    }
+    }
+    const {getUsers}= permissions;
+    //console.log(getUsers);
+    let f;
+    function hasPermission(moduleName,role,permissionType)
```

Comment:
Pass module name as sting instead on object.

Reply:
Sir, this was a test file it was added by mistake, correct file has already been pushed to develop .
sorry for the mistake sir